arm-redis/2016-04-01/swagger/redis.json

```diff
@@ -545,6 +545,213 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cache/Redis/{cacheName}/firewallRules": {
+      "get": {
+        "tags": [
+          "Redis",
+          "FirewallRules"
+        ],
+        "operationId": "RedisFirewallRules_List",
```

Comment:
@veronicagg The naming guidance seems a little arbitrary. E.g. We call the create operation RedisFirewallRule_CreateOrUpdate and there's no error for that? (Or I overlooked it)